app/controllers/api/v2/applications_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::V2::ApplicationsController < ActionController::API
+  def index
+    render json: { "applications": applications }
+  end
+
+  def show
+    if matching_application.present?
+      render json: matching_application
+    else
+      head :not_found
+    end
+  end
+
+private
+
+  def matching_application
```

Comment:
On Find we've been following the pattern of `build_application` (or `create_application` for creating). I'll leave it up to you if you want to follow that, it's a common pattern I've seen.

Reply:
Yeah, `build_` seems pretty idiomatic for this, good call.

Reply:
Addressed in 5bb745e